spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/storage/queue/factory/DefaultStorageQueueFactory.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+
+package com.microsoft.azure.spring.integration.storage.queue.factory;
+
+import com.microsoft.azure.management.storage.StorageAccount;
+import com.microsoft.azure.spring.cloud.context.core.AzureAdmin;
+import com.microsoft.azure.spring.cloud.context.core.AzureUtil;
+import com.microsoft.azure.spring.integration.core.Memoizer;
+import com.microsoft.azure.spring.integration.storage.queue.StorageQueueRuntimeException;
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueClient;
+
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+import java.util.UUID;
+import java.util.function.Function;
+
+public class DefaultStorageQueueFactory implements StorageQueueFactory {
```

Comment:
I'm confusing in your change about where should be queue client and real queue. Please make it clear.